### PR TITLE
Toc improvemetns

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

File: docs/workforce-management-framework/upstream-1/master.adoc [hunk 1]
```diff
@@ -47,18 +43,23 @@ include::topics/con-datasync-module.adoc[leveloffset=+2]
...
 include::topics/pro-keycloak-enablement.adoc[leveloffset=+2]
 include::topics/pro-keycloak-implementation.adoc[leveloffset=+2]
 
+== Security Passport.js
 include::topics/con-passportauth-module.adoc[leveloffset=+2]
 include::topics/con-passportauth-securitystrategy.adoc[leveloffset=+2]
 include::topics/ref-passportauth-securitystrategy.adoc[leveloffset=+3]
 include::topics/pro-passportauth-pointing-to-a-datasource.adoc[leveloffset=+2]
 
+include::topics/con-logging.adoc[leveloffset=+1]
```
Comment (JameelB) [inline docs/workforce-management-framework/upstream-1/master.adoc]: This is under the security heading.

File: docs/workforce-management-framework/upstream-1/master.adoc [hunk 2]
```diff
@@ -47,18 +43,23 @@ include::topics/con-datasync-module.adoc[leveloffset=+2]
 
 == Security
 include::topics/con-introducing-securityfeatures.adoc[leveloffset=+2]
+
+== Security {Keycloak}
```
Comment (JameelB) [inline docs/workforce-management-framework/upstream-1/master.adoc]: [Optional] Not sure about `Security {Keycloak}`, it doesn't sound right to me. I think just `Keycloak` is fine?

File: docs/workforce-management-framework/upstream-1/master.adoc [hunk 3]
```diff
@@ -47,18 +43,23 @@ include::topics/con-datasync-module.adoc[leveloffset=+2]
...
 include::topics/con-keycloak.adoc[leveloffset=+2]
 include::topics/ref-keycloak.adoc[leveloffset=+3]
 include::topics/con-keycloak-securitystrategy.adoc[leveloffset=+2]
 include::topics/ref-keycloak-securitystrategy.adoc[leveloffset=+3]
 include::topics/pro-keycloak-enablement.adoc[leveloffset=+2]
 include::topics/pro-keycloak-implementation.adoc[leveloffset=+2]
 
+== Security Passport.js
```
Comment (JameelB) [inline docs/workforce-management-framework/upstream-1/master.adoc]: ^ Same as above

File: docs/workforce-management-framework/upstream-1/master.adoc (hunk 1 above)
Comment (wtrocki) [reply docs/workforce-management-framework/upstream-1/master.adoc]: screen shot 2017-10-11 at 1 36 44 pm

**Review by witmicko — APPROVED** [review]
lgtm

**Comment by pwright** [comment]
@wtrocki  as discussed 
* Introducing RC Security Features
* Using Keycloak with RC 
* Using Passport with RC
You can have sub headings under each of the headings if needed.
You could replace "Introducing <something>" with "<something> Overview" if you feel that reads better, but be consistent
"Pointing Passport to a datasource"  => "Associating Passport with a Datasource"

**Comment by wtrocki** [comment]
@finp - Applied suggestions. 
It looks like other places will need to have this as improvements as well.
Let's review docs now as we have 99% of the content there.
We will also create meeting to go thru docs for final review.

**Pull request merged**